sdk/tests/tutorials/ibor/test_cut_labels.py

```diff
@@ -0,0 +1,231 @@
+import unittest
+from datetime import datetime, date, timedelta
+
+import pytz
+import uuid
+
+import lusid
+import lusid.models as models
+
+from lusid.utilities.api_client_builder import ApiClientBuilder
+from utilities.instrument_loader import InstrumentLoader
+from utilities.test_data_utilities import TestDataUtilities
+from utilities.credentials_source import CredentialsSource
+
+
+class CutLabels(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # create a configured API client
+        api_client = ApiClientBuilder().build(CredentialsSource.secrets_path())
+
+        cls.scopes_api = lusid.ScopesApi(api_client)
+        cls.portfolios_api = lusid.PortfoliosApi(api_client)
+        cls.transaction_portfolios_api = lusid.TransactionPortfoliosApi(api_client)
+        cls.property_definitions_api = lusid.PropertyDefinitionsApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+
+        cls.portfolios = lusid.PortfoliosApi(api_client)
+
+        cls.test_data_utilities = TestDataUtilities(cls.transaction_portfolios_api)
+
+        cls.cut_labels = lusid.CutLabelDefinitionsApi(api_client)
+
+    def test_cut_labels(self):
+        
+        def get_guid():
+            return str(uuid.uuid4())[:4]
+
+        # define function to format cut labels
+        def cut_label_formatter(date, cut_label_code):
+            return str(date) + "N" + cut_label_code
+        
+        
+        ## Create cut labels
+        # create a dictionary to keep track of cut labels
+        code = {}
+
+        def create_cut_label(hours, minutes, display_name, description, time_zone):
```

Comment:
I'm not sure if it was the same in your notebook, if so I missed it, but you should avoid referring to global variables inside a function. Here you refer to "code" inside your function but you don't pass it in as a parameter. Either add each cut label to the dictionary outside the function or pass it into your function and return it.

Reply:
Okay, I've added it as a parameter. Just checked my notebook and it's the same there, shall I change this as well?

Reply:
Sure, but not a priority, in the notebook less focus on the code